tests/qos/test_qos_sai.py

```diff
@@ -1016,6 +1070,15 @@ def testQosSaiDscpQueueMapping(
         if separated_dscp_to_tc_map_on_uplink(duthost, dut_qos_maps):
             pytest.skip("Skip this test since separated DSCP_TO_TC_MAP is applied")
 
+        srcport = dutConfig["testPorts"]["src_port_id"]
```

Comment:
@XuChen-MSFT could we use a function here, so many repeated code.

Reply:
add non-existent port correction behaviror into common wrapper function